packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -216,6 +216,24 @@
                             </oui-action-menu-item>
                         </oui-action-menu>
                     </oui-tile-definition>
+                    <!-- email offer -->
+                    <oui-tile-definition data-term="{{ :: 'hosting_dashboard_service_email_address' | translate}}">
+                        <oui-tile-description>
+                            <span data-ng-if="!$ctrl.isRetrievingEmailOffer"
+                                data-ng-bind="$ctrl.doesEmailOfferExists()  ? ('hosting_dashboard_service_common_active' | translate) : ('hosting_dashboard_service_common_inactive' | translate)"></span>
+                            <oui-spinner data-ng-if="$ctrl.isRetrievingEmailOffer"></oui-spinner>
+                        </oui-tile-description>
+                        <oui-action-menu
+                            data-compact data-align="end"
+                            data-ng-if="!$ctrl.isRetrievingEmailOffer"
+                            data-disabled="$ctrl.doesEmailOfferExists()">
+                            <oui-action-menu-item data-text="{{ ::'hosting_dashboard_service_activate_email' | translate }}"
+                                data-on-click="$ctrl.activateEmailOffer()"
+                                data-ng-if="!$ctrl.doesEmailOfferExists()">
```

Comment:
If there's an email offer, it looks like an empty action-menu will be displayed, maybe the `ng-if` should be on the parent instead